Task-6/Task-6/Models/OperatingSystemTypeEnum.cs

```diff
@@ -0,0 +1,10 @@
+namespace Task_6.Models;
+
+public class OperatingSystemEnum
+{
+    public enum OperatingSystemType
+    {
+        IOS,
+        Android
+    }
+}
```

Comment:
Incorrect enum. Don't wrap enum in class, remove public class OperatingSystemEnum